src/components/catalogs/Catalogs.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext } from 'react';
+import { GlobalContext } from '../../reducer';
+import { Product } from '../../types/product';
+import { NavLinkCustom } from '../navLink/NavLinkCustom';
+import './catalogs.scss';
+
+export const Catalogs = () => {
+  const [state] = useContext(GlobalContext);
+
+  return (
+    <div className="wrapper-catalogs">
+      <NavLinkCustom
+        way="/phones"
+        classStyle="link"
+        data-cy="categoryLinksContainer"
+      >
+        <div className="catalog">
+          <img src="./img/banners/phone.png" alt="phones" />
+          <div className="catalog__info">
+            <span>Mobile phones</span>
+            <span>
+              {
+                state.catalogsProducts.filter(
+                  (el: Product) => el.type === 'phone',
+                ).length
+              }
+              {' '}
+              models
+            </span>
+          </div>
+        </div>
+      </NavLinkCustom>
+      <NavLinkCustom
+        way="/tablets"
+        classStyle="link"
+        data-cy="categoryLinksContainer"
+      >
+        <div className="catalog">
+          <img src="./img/banners/tablets.png" alt="tablets" />
+          <div className="catalog__info">
+            <span>Tablets</span>
+            <span>
+              {
+                state.catalogsProducts.filter(
+                  (el: Product) => el.type === 'tablet',
```

Comment:
if you type `state.catalogsProducts` correctly, this `el: Product` is redundant

Reply:
+